Geometry_Engine/Query/Bounds.cs

```diff
@@ -386,6 +387,27 @@ public static BoundingBox Bounds(this List<Point> pts)
 
         /***************************************************/
 
+        public static BoundingBox Bounds(this List<BoundingBox> boxes)
```

Comment:
If we want to keep this, we can just use the operator `+` between `BoundingBox`es maybe?

Reply:
If you'd like, I just went straight for the definition, (mostly for the aesthetics of avoiding assigning the first value and then looping over the rest of them, but maybe silly)